.github/workflows/commit-to-main.yml

```diff
@@ -0,0 +1,87 @@
+name: Trigger basic downstream CI
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  trigger-downstream-ci:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      # TODO: missing projects?
+      - name: Trigger OQS-OpenSSL CI
+        run: |
+          curl --silent \
+               --write-out "\n%{response_code}\n" \
+               --user ${BUILD_TRIGGER_TOKEN}: \
+               --request POST \
+               --header "Content-Type: application/json" \
+               --data '{ "branch": "OQS-OpenSSL_1_1_1-stable", "parameters": { "run_downstream_tests": true } }' \
+               https://circleci.com/api/v2/project/gh/open-quantum-safe/openssl/pipeline | tee curl_out \
+          && grep -q "201" curl_out
+      - name: Trigger OQS-BoringSSL CI
+        run: |
+          curl --silent \
+               --write-out "\n%{response_code}\n" \
+               --user ${BUILD_TRIGGER_TOKEN}: \
+               --request POST \
+               --header "Content-Type: application/json" \
+               --data '{ "branch": "master", "parameters": { "run_downstream_tests": true } }' \
+               https://circleci.com/api/v2/project/gh/open-quantum-safe/boringssl/pipeline | tee curl_out \
+          && grep -q "201" curl_out
+      - name: Trigger OQS-OpenSSH CI
+        run: |
+          curl --silent \
+               --write-out "\n%{response_code}\n" \
+               --user ${BUILD_TRIGGER_TOKEN}: \
+               --request POST \
+               --header "Content-Type: application/json" \
+               --data '{ "branch": "OQS-v8", "parameters": { "run_downstream_tests": true } }' \
```

Comment:
When to add/change this to v9?

Reply:
The v9 branch doesn't yet work; the [current PR from @geedo0](https://github.com/open-quantum-safe/openssh/pull/160) gets the basic test suite running but still doesn't have full PQ or hybrid support.  I think we can merge this PR with v8, and then when v9 is ready, we can come back and update the liboqs CI.